diff --git a/rtl/chip/SA1/SA1Map.vhd b/rtl/chip/SA1/SA1Map.vhd
index fca68aa..23c8fa2 100644
--- a/rtl/chip/SA1/SA1Map.vhd
+++ b/rtl/chip/SA1/SA1Map.vhd
@@ -11,7 +11,8 @@ entity SA1Map is
 		MCLK			: in std_logic;
 		RST_N			: in std_logic;
 		ENABLE		: in std_logic := '1';
-		
+		PAUSE			: in std_logic := '0';
+
 		CA   			: in std_logic_vector(23 downto 0);
 		DI				: in std_logic_vector(7 downto 0);
 		DO				: out std_logic_vector(7 downto 0);
@@ -81,6 +82,7 @@ begin
 		CLK			=> MCLK,
 		RST_N			=> RST_N and MAP_SEL,
 		ENABLE		=> ENABLE,
+		PAUSE			=> PAUSE,
 
 		SNES_A		=> CA,
 		SNES_DO		=> DO,
